Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -607,6 +608,11 @@ namespace AzToolsFramework
             }
         }
 
+        bool IsFolderEmpty(const char* pPath)
```

Comment:
update: Please don't use hungarian notation (`p` prefix) - it does crop up in older cry code and can make sense to match style there but in AZ code we should try and avoid it

Reply:
update: Passing a `QString` here or an `AZStd::string_view` might also be better as then we won't have to call `strlen` when creating `QDir`

Reply:
Also if this is just used in the .cpp it can be marked static to give it internal linkage (and the ability to be inlined more easily)